src/UiCommand.php

```diff
@@ -41,11 +41,11 @@ public function handle()
             throw new InvalidArgumentException('Invalid preset.');
         }
 
-        $this->{$this->argument('type')}();
-
         if ($this->option('auth')) {
             $this->call('ui:auth');
         }
+
+        $this->{$this->argument('type')}();
```

Comment:
Note that the order is changed here because the React preset needs to change a file provided by the auth command.